spec/factories/unavailabilities.rb

```diff
@@ -1,5 +1,12 @@
 FactoryGirl.define do
   factory :unavailability do
```

Comment:
Better to put it in the spec, for two reasons:

1. You don't need to read two different files to determine why a test is passing or failing;
2. A factory should only have the absolute minimum information required to create a valid instance of an object (that is, it should echo the model validations).